### PR TITLE
amp-layout: enable visual tests

### PR DESCRIPTION
**Pull request by aghassemi**

### CODE REVIEW

**Comment by aghassemi** [comment]
@rsimha-amp, my component (`amp-layout`) is not in production yet and it looks like visual tests use production binaries to take screenshots. Is the solution to wait for the component to launch before adding visual tests? (which is definitely fine, just wanted to see if there are other options)

**Comment by rsimha-amp** [comment]
You can run the test against canary. Would that help?

**Comment by aghassemi** [comment]
@rsimha-amp it's not even in Canary yet. I would need to run them against `master` to be able to approve the screenshots now.

File: test/visual-diff/visual-tests.json [hunk 1]
```diff
@@ -107,6 +107,10 @@
         ".comic-amp-font-loaded",
         ".comic-amp-bold-font-loaded"
       ]
+    },
+    {
+      "url": "examples/visual-tests/amp-layout/amp-layout.amp.html",
+      "name": "AMP Layout"
```
Comment (rsimha-amp) [inline test/visual-diff/visual-tests.json]: I assume there are no css elements to wait for, or special experiments to enable here? (If so, you can do so by adding a `loading-complete-css` or `experiments` section here, as seen in the example at the top of this file.)

**Comment by rsimha-amp** [comment]
@aghassemi Instructions for the next steps you must take are in the travis logs for this PR. See https://travis-ci.org/ampproject/amphtml/jobs/305517899#L696 and let me know if you have any questions.

**Comment by aghassemi** [comment]
Got it. Thanks!

**Comment by rsimha-amp** [comment]
You're right, these look different on chrome when I test locally. Perhaps phantomjs (used for percy snapshots) doesn't support block highlighting?

**Pull request merged**